example_config.toml

```
@@ -14,10 +14,14 @@ network = "regtest"
 
 [sync]
 l1_follow_distance = 6
-max_reorg_depth = 4
-client_poll_dur_ms = 200
 client_checkpoint_interval = 10
 
+[btcio]
+client_poll_dur_ms = 200
+write_poll_dur_ms = 200
+fee_policy = "Smart"
```

Maybe we can do a `#[serde(rename_all = "lowercase")]` in the `enum` above so that this can become:

```suggestion
fee_policy = "smart"
```

See: https://serde.rs/container-attrs.html